virttest/libvirt_xml/vm_xml.py

```diff
@@ -1783,29 +1777,196 @@ def __init__(self, virsh_instance=base.virsh):
         super(VMCPUXML, self).__init__(virsh_instance=virsh_instance)
         self.xml = '<cpu/>'
 
-    @staticmethod
-    def marshal_from_cell(item, index, libvirtxml):
+    # Sub-element of cpu/numa
+    class Numa_CellXML(VMXML):
```

Comment:
Looks like this is not the usually way we try to handle a nested xml, may I ask the reason of that? 
According to our usually way, we could change 'numa_cell' from 'XMLElementList' to 'XMLElementNest' if the 'list' type of element no longer fits our need.  I suppose that 'numa_cell' element will still be nested under 'cpu', then wouldn't it make more sense to just use the old way of setting to 'XMLElementNest'? Not mention you have already setup the xml class.

Reply:
Since there at least two levels' list, the usual way doesn't work well as I ever tried. And because it is a list , the "XMLElementNest" does not match aslo.